hasura/migrations/default/1654202757203_alter_table_public_circle_api_keys_add_column_read_discord/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- alter table "public"."circle_api_keys" add column "read_discord" boolean
+--  not null default 'false';
```

Comment:
let's delete this boilerplate.

Reply:
Added a down migration with https://github.com/coordinape/coordinape/pull/1625/commits/c4056e70f74ae6a8678949429e777757c3e487ea